File: Ananya Chandra/calculator.py
def add(x,y):
    return(x+y)

def sub(x,y):
    return(x-y)

def mutiply(x,y):
    return(x*y)

def division(x,y):
    return(x/y)

print(" welcome to python calculator")
print("+ for addition")
print("- for subtraction")
print("* for multiplication")
print("/ for division")
choice = input(" enter the operator")
num1=int(input(" first number"))
num2=int(input(" second number"))

if choice=="+":
    print(add(num1,num2))
if choice=="-":
    print(sub(num1,num2))
if choice=="*":
    print(mutiply(num1,num2))
if choice=="/":
    print(division(num1,num2))
else:
    print("invalid choice")

File: ShreyanshSh31/calculator.java
import java.io.*;
class calculator extends pattern_fourth
{
    public static void main(String args[]) throws IOException
    {
        double a,b,result;
        result=0;
        int choice;
        InputStreamReader reader = new InputStreamReader(System.in);
        BufferedReader input= new BufferedReader(reader);
        System.out.print("Enter First Number::");
        String v1= input.readLine();
        a= Double.parseDouble(v1);
        System.out.println("Enter the second Number::");
        String v2 =input.readLine();
        b=Double.parseDouble(v2);
        System.out.println("Enter operator(+,-,*,/)::");
        System.out.println("Enter 1 for addition +");
        System.out.println("Enter 2 for substraction -");
        System.out.println("Enter 3 for Multiplication *");
        System.out.println("Enter 4 for division /");
        String n = input.readLine();
        choice= Integer.parseInt(n);
        if(choice==1)
        {
            result = a+b;
            System.out.println("Result ="+result);
        }
        else if (choice==2){

            result = a-b;
            System.out.println("Result ="+result);
        }
        else if (choice==3)
        {
            result= a*b;
            System.out.println("Result ="+result);
        }
        else if (choice==4){
            result=a/b;
            System.out.println("Result ="+result);
        }
        else 
            System.out.println("It is a wrong choice");
    }
}
